keps/sig-storage/1491-csi-migration-vsphere/kep.yaml

```
@@ -0,0 +1,54 @@
+title: In-tree Storage Plugin to CSI Migration - vSphere
+kep-number: 1491
+authors:
+  - "@divyenpatel"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-cluster-lifecycle
+reviewers:
+  - "@Jiawei0227"
+  - "@msau42"
+approvers:
+  - "@msau42"
+editor: "@Jiawei0227"
+creation-date: 2022-01-11
+last-updated: 2022-01-11
+disable-supported: true
+status: implementable
+see-also:
+  - "https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md"
+prr-approvers:
+  - "@wojtek-t"
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.24"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.18"
+  beta: "v1.19"
```

```suggestion
  alpha: "v1.19"
  beta: "v1.24"
```

per implementation history above

---

Sorry I made a mistake in the implementation history, it should be 1.18 and 1.19